blocks/leds.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license
+ * Visual Blocks Editor
+ *
+ * opyright 2012 Google Inc.
+ * Copyright 2019 Rovercode
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Motor blocks for Blockly.
+ * @author brady.hurlburt@rovercode.com (Brady L. Hurlburt)
+ */
+'use strict';
+
+goog.provide('Blockly.Blocks.leds');
+
+goog.require('Blockly.Blocks');
+
+
+/**
+ * Common HSV hue for all blocks in this category.
+ */
+Blockly.Blocks.leds.HUE = 77;
+
+Blockly.Blocks['chainable_rgb_led_set'] = {
+  init: function() {
+    this.appendDummyInput()
+        .appendField("set LED ");
+    this.appendValueInput("LED_ID")
+        .setCheck("Number");
+    this.appendDummyInput()
+        .appendField("to colour ");
```

Comment:
British `colour`?

Reply:
Ah, good catch. On the code side, the `setCheck` right below this expects `COLOUR`, so I figured I'd be consistent.
But this field is user-facing, so I'll change it to `color`.

Reply:
Actually, the existing blocks that we'll use for picking colors are spelled `colour`, too, so I think I'd like to leave this.

Reply:
![image](https://user-images.githubusercontent.com/976857/63310347-4ea20700-c2c8-11e9-811a-3ebd4d196878.png)